Jenkinsfile

```diff
@@ -8,6 +8,32 @@ pipeline {
     }
     
     stages {
+        stage('Generate Assets') {
+            options {
+                timeout(time: 20)
```

Comment:
Now that we are extracting assets once and stashing them, we can probably remove this timeout from the other stages. That was added due to a rare bug where extraction would hang up (haven't seen it for a while, fixed now?) or get stuck in an infinite loop (saw once). It should help with stages timing out when multiple jobs are queued, as well.

Alternatively, we could just set the timeouts on the other stages to a much higher value, like 60 min or something.

Reply:
It will not timeout unless explicitly set.